ci/code_checks.sh

```diff
@@ -203,10 +203,11 @@ if [[ -z "$CHECK" || "$CHECK" == "code" ]]; then
 import sys
 import pandas
 
-blacklist = {'bs4', 'gcsfs', 'html5lib', 'ipython', 'jinja2' 'hypothesis',
+blacklist = {'bs4', 'gcsfs', 'html5lib', 'http', 'ipython', 'jinja2' 'hypothesis',
              'lxml', 'numexpr', 'openpyxl', 'py', 'pytest', 's3fs', 'scipy',
-             'tables', 'xlrd', 'xlsxwriter', 'xlwt'}
-mods = blacklist & set(m.split('.')[0] for m in sys.modules)
+             'tables', 'urllib.request', 'xlrd', 'xlsxwriter', 'xlwt'}
+import_mods = set(m.split('.')[0] for m in sys.modules) | set(sys.modules)
```

Comment:
why did this need to change?

Reply:
without this we couldn't blacklist `urllib.request`, only `urrlib` (which we dont want to blacklist)

Reply:
ok, can you add a comment, this is not obvious